website/templates/project/modal_add_contributor.mako

```diff
@@ -272,3 +272,5 @@
     </div><!-- end modal-dialog -->
 </div><!-- end modal -->
 
+<link href="/static/css/add-contributors.css" rel="stylesheet">
```

Comment:
Can you `require` this (in the JS module for the modal) instead of using a `link` tag?